processing/src/main/java/org/apache/druid/segment/join/PostJoinCursor.java

```diff
@@ -99,15 +108,15 @@ public Filter getPostJoinFilter()
   @Override
   public void advance()
   {
-    advanceUninterruptibly();
-    BaseQuery.checkInterrupted();
```

Comment:
Can we add a note on why the checkInterrupted() is not needed anymore

Reply:
Added a comment.